functions/packages.bash

```diff
@@ -404,29 +378,23 @@ The article also contains instructions regarding openHAB integration.
     if ! (whiptail --title "Description, Continue?" --yes-button "Continue" --no-button "Back" --yesno "$introtext" 15 80) then echo "CANCELED"; return 0; fi
   fi
 
-  cond_redirect apt-get update
-  cond_redirect apt-get -y install git python3 python3-pip bluetooth bluez
-  if [ $? -ne 0 ]; then echo "FAILED (prerequisites)"; exit 1; fi
+  if ! cond_redirect apt-get -y install git python3 python3-pip bluetooth bluez; then echo "FAILED (prerequisites)"; exit 1; fi
   if [ ! -d "$DIRECTORY" ]; then
     cond_echo "Fresh Installation... "
     cond_redirect git clone https://github.com/ThomDietrich/miflora-mqtt-daemon.git $DIRECTORY
-    cond_redirect cp $DIRECTORY/config.{ini.dist,ini}
-    if [ $? -ne 0 ]; then echo "FAILED (git clone)"; exit 1; fi
+    if ! cond_redirect cp $DIRECTORY/config.{ini.dist,ini}; then echo "FAILED (git clone)"; exit 1; fi
   else
     cond_echo "Update... "
-    cond_redirect git -C $DIRECTORY pull --quiet origin
-    if [ $? -ne 0 ]; then echo "FAILED (git pull)"; exit 1; fi
+    if ! cond_redirect git -C $DIRECTORY pull --quiet origin; then echo "FAILED (git pull)"; exit 1; fi
```

Comment:
see up

Reply:
@mstormi

Reply:
This is two-issues-in-one but  once I changed the SC2181 issue, shellcheck actually does NOT request the quotation marks to be present any more so I didn't change it, that's what my comment was referring to. Seems that was misunderstood.
To avoid further discussion, I now added them but also added them in all of the next 5 lines as it would be sort of inconsistent to not do that as well.